lib/prepare.js

```diff
@@ -43,9 +45,16 @@ const CDV_ANY_SIZE_CLASS = 'any';
 module.exports.prepare = function (cordovaProject, options) {
     const platformJson = PlatformJson.load(this.locations.root, 'ios');
     const munger = new PlatformMunger('ios', this.locations.root, platformJson, new PluginInfoProvider());
-
     this._config = updateConfigFile(cordovaProject.projectConfig, munger, this.locations);
 
+    const parser = new ConfigParseriOS(cordovaProject.projectConfig.path);
```

Comment:
If class name changes, update here as well.

Reply:
Thanks, I updated it.